examples/sagemaker-tensorflow/sagemaker-merlin-tensorflow.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>better to explain <code>local machine</code>  I assume you meant a machine with a compatible GPU? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/692/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>